### PR TITLE
[ GetGameTitleById ] created

### PR DESCRIPTION
**Pull request by andreamir**
😉

### CODE REVIEW

File: src/api/gameTitles/gameTitles.model.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Schema, model } from 'mongoose';
+
+const gameTitlesSchema = new Schema({
+  title: {
+    type: String,
+  },
```
Comment (nuinn) [inline src/api/gameTitles/gameTitles.model.js:8]: I think title should be required and unique

**Review by nuinn — CHANGES_REQUESTED** [review]

File: src/api/gameTitles/gameTitles.model.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Schema, model } from 'mongoose';
+
+const gameTitlesSchema = new Schema({
+  title: {
+    type: String,
```
Comment (jaanmeld) [inline src/api/gameTitles/gameTitles.model.js:5]: Posiiblemente tenia que ser required?

Comment (jaanmeld) [reply src/api/gameTitles/gameTitles.model.js:5]: Title

File: src/api/gameTitles/gameTitles.model.js (hunk 1 above)
Comment (andreamir) [reply src/api/gameTitles/gameTitles.model.js:8]: done ✔︎

**Review by jaanmeld — CHANGES_REQUESTED** [review]

**Review by nuinn — APPROVED** [review]
LGTM

**Review by Ahuanmarca — APPROVED** [review]

**Review by jaanmeld — APPROVED** [review]

**Review by JonaTheBridge — APPROVED** [review]

**Pull request merged**